### PR TITLE
Add specifying auth_token in a different file

### PR DESCRIPTION
**Pull request by igorlistopad**

### CODE REVIEW

File: lib/util/settings.js [hunk 1]
```diff
@@ -314,7 +314,12 @@ function write() {
     const actual = yaml.read(file);
 
     // In case the setting is defined in a separte file (e.g. !secret network_key) update it there.
-    for (const path of [['mqtt', 'user'], ['mqtt', 'password'], ['advanced', 'network_key']]) {
+    for (const path of [
+        ['mqtt', 'user'],
+        ['mqtt', 'password'],
+        ['advanced', 'network_key'],
+        ['frontend', 'auth_token']
```
Comment (Koenkk) [inline lib/util/settings.js:321]: Please add a trailing comma here.

**Comment by Koenkk** [comment]
Have you tried if this actually works? I expect something similar like this needs to be done: https://github.com/Koenkk/zigbee2mqtt/blob/cec1c958f1a7849fcb0ab8ccec330e952d014c5f/lib/util/settings.js#L438

File: lib/util/settings.js (hunk 1 above)
Comment (igo-r) [reply lib/util/settings.js]: Done

**Comment by igo-r** [comment]
> Have you tried if this actually works? I expect something similar like this needs to be done:
> 
> https://github.com/Koenkk/zigbee2mqtt/blob/cec1c958f1a7849fcb0ab8ccec330e952d014c5f/lib/util/settings.js#L438

Right, I transferred it from a local installation in the dead of night, I forgot to add changes to this function.

It all right now!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**